spec/support/fakeweb.rb

```diff
@@ -2,13 +2,17 @@
 FakeWeb.allow_net_connect = false
 
 def fake_post path, response
-  FakeWeb.register_uri(:post, "https://pi.pardot.com#{path}", :body => response)
+  FakeWeb.register_uri(:post, "https://pi.pardot.com#{path}", :body => response, :Authorization => 'user_key=bar&api_key=my_api_key')
 end
 
 def fake_get path, response
-  FakeWeb.register_uri(:get, "https://pi.pardot.com#{path}", :body => response)
+  FakeWeb.register_uri(:get, "https://pi.pardot.com#{path}", :body => response, :Authorization => 'user_key=bar&api_key=my_api_key')
```

Comment:
Shouldn't these have `Pardot` at the front of the string?

Reply:
FakeWeb doesn't support matching based on Authorization so these values weren't being used. I've removed them in the latest.